core/src/main/java/io/questdb/griffin/engine/table/LatestByValueIndexedFilteredRecordCursor.java

```diff
@@ -36,8 +37,10 @@ class LatestByValueIndexedFilteredRecordCursor extends AbstractDataFrameRecordCu
     protected final int columnIndex;
     protected final int symbolKey;
     private final Function filter;
+    private SqlExecutionCircuitBreaker circuitBreaker;
     private boolean found;
     private boolean hasNext;
```

Comment:
nit: having found and isRecordFound fields is a bit confusing, especially that found actually means that record was found, while isRecordFound - that findRecord() was called .

Reply:
Renamed to `isFindPending` and `isRecordFound` in 54715671daf89c7f764bd93bc3225e5618f6afb8